tests/acceptance/stepDefinitions/filesContext.js

```diff
@@ -121,10 +121,15 @@ When('the user deletes the following elements using the webUI', function (table)
   return client.page.filesPage()
 })
 
-Given('the following files have been deleted by user {string}', function (user, table) {
-  for (const line of table.rows()) {
-    webdav.delete(user, line[0])
-  }
+Given('the following files have been deleted by user {string}', async function (user, table) {
+  const filesToDelete = table.rows()
+  await Promise.all(
```

Comment:
As we send multiple requests at once, this might fail for some stupid scenarios we want to test. 
Eg:
```feature
Given the following files have been deleted by user "user0"
 | simple-folder           |
 | simple-folder/lorem.txt |
```
As the folder will be locked, anything might happen. `lorem.txt` might get deleted first, and then the folder itself, and `simple-folder` might get deleted and get locked.